CRM/Contact/Form/Task.php

```diff
@@ -88,7 +88,7 @@ public static function preProcessCommon(&$form) {
 
     $isStandAlone = in_array('task', $form->urlPath) || in_array('standalone', $form->urlPath);
     if ($isStandAlone) {
-      list($form->_task, $title) = CRM_Contact_Task::getTaskAndTitleByClass(get_class($form));
+      [$form->_task, $title] = CRM_Contact_Task::getTaskAndTitleByClass(get_class($form));
```

Comment:
Cool. I didn't know about the square bracket shortcut for `list()`. Learn something new every day around here...

Reply:
yeah phpstorm keeps making that change on me :-)